packages/spacecat-shared-utils/src/formcalc.js

```diff
@@ -11,7 +11,9 @@
  */
 
 const DAILY_PAGEVIEW_THRESHOLD = 200;
-const CR_THRESHOLD_RATIO = 0.2;
+const CR_THRESHOLD_RATIO = 0.3;
+//audit interval
+const INTERVAL = 15;
```

Comment:
Let's update the default value of the methods to consume interval from here.
And then we can remove the arg from worker and reporter